tutorials/super_blink/blink_config.h

```diff
@@ -0,0 +1,31 @@
+#include "libs/can/api.h"
+#include "libs/gpio/api.h"
+#include "libs/gpio/pin_defs.h"
+#include "libs/timer/api.h"
+
+void timer0_callback(void);
+
+gpio_t LED1 = PB0;
+gpio_t LED2 = PB1;
+
+timer_cfg_s timer0_cfg = {
+    .timer = TIMER0,
+    .timer0_mode = TIMER0_MODE_CTC,
+    .prescalar = CLKIO_DIV_64,
+    .channel_a = {
+        .output_compare_match = 0x822,
+        .pin_behavior = DISCONNECTED,
+        .interrupt_enable = true,
+        .interrupt_callback = timer0_callback,
+    },
+};
+
+// Define CAN message
+uint8_t OEM_TEXT[3] = { 79, 69, 77 };
```

Comment:
Technically, if you want, you could do

```c
uint8_t OEM_TEXT[3] = { 'O', 'E', 'M' };
```

If you have a letter or number in single quotes in C, it is a `char` type (short for character). In AVR, `char`s are 8-bits, so they are the same as `uint8_t`s. I don't _think_ you'll get a compiler warning if you do, but you might...